dev/devicelab/bin/tasks/new_gallery_chrome_run_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+Future<void> main() async {
+  await task(const NewGalleryChromeRunTest().run);
+}
+
+/// URI for the New Flutter Gallery repository.
+const String galleryRepo = 'https://github.com/flutter/gallery.git';
+
+/// After the gallery loads, a duration of [durationToWaitForError]
+/// is waited, allowing any possible exceptions to be thrown.
+const Duration durationToWaitForError = Duration(seconds: 5);
+
+/// Flutter prints this string when an app is successfully loaded.
+/// Used to check when the app is successfully loaded.
+const String successfullyLoadedString = 'To hot restart';
+
+/// Flutter prints this string when an exception is caught.
+/// Used to check if there are any exceptions.
+const String exceptionString = 'EXCEPTION CAUGHT';
+
+/// Checks that the New Flutter Gallery runs successfully on Chrome.
```

Comment:
Are you looking for compilation success or no errors at startup?

Reply:
I'm checking that there are no errors during the 5 seconds when the app is running.